uxarray/utils/jit_attributes.py

```diff
@@ -0,0 +1,2 @@
+cache = True
+jit_bool = False
```

Comment:
I think we don't need a new file, this can go into constants.py

Reply:
Just a note: Our use case here was to study performance, in regular usage one might never turn off caching as it will only help if we frequently modifying the code or have very limited disk space.

Reply:
Do you want me to remove the cache=True attribute then?

Reply:
> Do you want me to remove the cache=True attribute then?

No, let it be, just the above in a comment and a more detailed comment near the definition